### ISSUE TITLE
[Adaptive Card iOS - Flight Update]: Voice Over user does not able to navigate to headings using swipe up/down gesture after changing rotor setting into heading mode.  

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#28]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'FlightUpdate.JSON' and double tap to activate it.
7. The Card will open.
8. Change rotar settings to heading.
9. Navigate using up/down swipe and observe the issue.

**Actual Result:**
Voice Over user does not able to navigate on the headings present under Flight Update Card using Up/Down swipe gesture after changing rotor setting to heading mode.

**Expected Result:**
Voice Over user should be able to navigate on the headings present under card using Up/Down arrow swipe action.

**Observations:**
This issue is not repro while navigating using left/right swipe gesture.

**User Impact:**
Low vision person who uses screen reader will not navigate to heading to heading for fast navigation on screen.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/747cdb29-cd7a-46d9-80b2-11b21322d402
